libs/scripts/release.sh

```diff
@@ -60,10 +57,13 @@ function bump-npm () {
     git reset --hard ${GIT_TAG}
 
     # grab change log early, before the version bump
-    CHANGE_LOG=$(git-changelog)
+    LAST_RELEASED_SHA=$(jq -r '.audius."last-released-sha"' package.json)
```

Comment:
by convention js is camel case so this should be like lastReleasedSHA

Reply:
@dmanjunath sounds good! 

I was matching package.json's convention that uses plenty with are hypenated-keys and only one instance of camelCasing: https://github.com/AudiusProject/audius-protocol/blob/b39fcc9ae51766ff9622391a936aaf6d967c43ae/libs/package.json#L78

I'll make the change to use camelCasing for this PR and subsequent package.json edits.

Reply:
thanks for changing. you want to drop the last-? i'm not too hung up on the exact property name so whatever you want to use is fine